### PR TITLE
test version

### PR DESCRIPTION
**Pull request by NikolayKrishchuk**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -59,3 +59,6 @@ typings/
 
 # next.js build output
 .next
+
+# IDE files: JetBrains
+.idea
```
Comment (meodai) [inline .gitignore:64]: can you remove this from here? you should have this in your global git ignore.

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  access_token: process.env.ACCESS_TOKEN,
+  access_token_secret: process.env.ACCESS_TOKEN_SECRET,
+});
+
+const sendRandomImage = require('./sendRandomImage');
+
+
+setInterval(() => {
+  sendRandomImage(T);
+}, 30000);
```
Comment (meodai) [inline src/index.js]: so it tweets every 30 minutes?

File: src/lib/generateImage.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  }
+
+  const canvas = createCanvas(300, 300);
+  const ctx = canvas.getContext('2d');
+
+  ctx.fillStyle = `rgb(${rgbBackground.r},${rgbBackground.g},${rgbBackground.b})`;
+  ctx.fillRect(0, 0, 300, 300);
+
+  ctx.fillStyle = 'rgb(0,0,0)';
+  ctx.font = 'bold 25px Impact Bold';
```
Comment (meodai) [inline src/lib/generateImage.js]: could we have a custom font here? https://rsms.me/inter/

File: src/lib/t.js [hunk 4]
```diff
@@ -0,0 +1,3 @@
+const {hexToRgb} = require('color-name-list/scripts/lib');
+
+console.log(hexToRgb('#5ba8ff'))
```
Comment (meodai) [inline src/lib/t.js]: we should remove that

File: src/index.js [hunk 5]
```diff
@@ -10,23 +11,21 @@ const T = new Twit({
   access_token_secret: process.env.ACCESS_TOKEN_SECRET,
 });
 
-/*
+
 setInterval(() => {
   sendRandomImage(T).catch((e) => console.log(e));
-}, 60000);
-*/
+}, 60000 * 5);
```
Comment (meodai) [inline src/index.js:17]: ah ok, might be cool to have this in the package.json? Like botconfig: { postFrequency: 300000} for example

**Review by meodai — COMMENTED** [review]
could you comment the code a little bit? Similarly to: https://github.com/meodai/color-names/blob/master/scripts/lib.js Just say what the function does...

File: src/index.js (hunk 2 above)
Comment (NikolayKrishchuk) [reply src/index.js]: here will be env variable

File: src/index.js (hunk 5 above)
Comment (NikolayKrishchuk) [reply src/index.js:17]: here will be env variable

Comment (NikolayKrishchuk) [reply src/index.js:17]: or options.file where I can write (if process.env.RETRY_DELAY || 60000 * 5)...

File: src/lib/t.js (hunk 4 above)
Comment (NikolayKrishchuk) [reply src/lib/t.js]: you see old commit

File: src/lib/generateImage.js (hunk 3 above)
Comment (NikolayKrishchuk) [reply src/lib/generateImage.js]: I will try to add

File: src/index.js (hunk 5 above)
Comment (meodai) [reply src/index.js:17]: process.env is cool as well

File: src/lib/t.js (hunk 4 above)
Comment (meodai) [reply src/lib/t.js]: oh you might squash commits before doing a pull request

**Pull request merged**